### PR TITLE
Put system user fixes

### PR DESCRIPTION
**Pull request by naman108**
address existing issues with put system user endpoint along with the addition of a new getAll endpoint to access existing system users

### CODE REVIEW

File: FreeTAKServer/controllers/services/RestAPI.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 import qrcode
 import io
 
+from FreeTAKServer.model.SQLAlchemy.system_user import SystemUser
```
Comment (brothercorvo) [inline FreeTAKServer/controllers/services/RestAPI.py:19]: > __[Pylint]__ Unused SystemUser imported from FreeTAKServer.model.SQLAlchemy.system_user ([view](https://sider.review/gh/repos/238457011/pulls/446#issue-8355639))
>
> <details>
>
> | Rule | Severity | Message-id | Module | Obj |
> | ---- | -------- | ---------- | ------ | --- |
> | `unused-import` | `warning` | W0611 | FreeTAKServer.controllers.services.RestAPI | - |
>
> You can [close this issue](https://sider.review/gh/repos/238457011/pulls/446#issue-8355639) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Remove unuses import

File: FreeTAKServer/controllers/services/RestAPI.py [hunk 2]
```diff
@@ -288,9 +307,44 @@
         userjson["Uid"] = user.uid
         userjson["DeviceType"] = user.device_type
         jsondata["SystemUsers"].append(userjson)
+    return jsondata
+
+
+@socketio.on('systemUser')
+@socket_auth(session=session)
+def systemUsers(empty=None):
```
Comment (brothercorvo) [inline FreeTAKServer/controllers/services/RestAPI.py:315]: > __[Pylint]__ function already defined line 280 ([view](https://sider.review/gh/repos/238457011/pulls/446#issue-8355621))
>
> <details>
>
> | Rule | Severity | Message-id | Module | Obj |
> | ---- | -------- | ---------- | ------ | --- |
> | `function-redefined` | `error` | E0102 | FreeTAKServer.controllers.services.RestAPI | systemUsers |
>
> You can [close this issue](https://sider.review/gh/repos/238457011/pulls/446#issue-8355621) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Double  check this function: several issues associated

**Comment by brothercorvo** [comment]
Most remaining issues are related to REST API conflicts, @naman108 , please solve them and re-commit

File: FreeTAKServer/controllers/DatabaseControllers/table_controllers.py [hunk 3]
```diff
@@ -47,11 +52,13 @@ def query(self, session, query, columns):
...
-                setattr(dp, column, value)
-        session.commit()
-
+        if len(DataPackages) >0:
+            for dp in DataPackages:
+                for column, value in column_value.items():
+                    setattr(dp, column, value)
+            session.commit()
+        else:
+            raise ValueError(f"no database entries which meet filter criteria {query}")
```
Comment (dlc-ariel) [inline FreeTAKServer/controllers/DatabaseControllers/table_controllers.py:61]:
```suggestion
        if len(DataPackages) == 0:
            raise ValueError(f"no database entries which meet filter criteria {query}")
            return
        for dp in DataPackages:
            for column, value in column_value.items():
                setattr(dp, column, value)
        session.commit()  
```

Doesnt really apply here but to reduce cognitive complexity we can do the following.
Generally putting a loop inside an if statement raises complexity considerable when checked using static code analyzers.

Comment (naman108) [reply FreeTAKServer/controllers/DatabaseControllers/table_controllers.py:61]: that makes perfect sense

**Pull request merged**